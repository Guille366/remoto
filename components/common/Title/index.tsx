const Title: React.FC = ({ children }) => {
  return <h2 className="text-2xl">{children}</h2>;
};

export default Title;
