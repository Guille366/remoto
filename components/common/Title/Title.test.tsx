import { render } from "@testing-library/react";
import Title from ".";

describe('Loading', () => {
  it('renders properly', () => {
    const { container } = render(<Title>Test</Title>);

    expect(container.firstChild).toMatchSnapshot();
  });
}); 