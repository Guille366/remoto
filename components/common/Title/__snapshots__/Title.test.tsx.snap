// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading renders properly 1`] = `
<h2
  class="text-2xl font-nunito font-bold"
>
  Test
</h2>
`;
